cms/envs/common.py

```diff
@@ -2076,9 +2076,10 @@
 
 ############## Installed Django Apps #########################
 
-from openedx.core.djangoapps.plugins import plugin_apps, plugin_settings, constants as plugin_constants
-INSTALLED_APPS.extend(plugin_apps.get_apps(plugin_constants.ProjectType.CMS))
-plugin_settings.add_plugins(__name__, plugin_constants.ProjectType.CMS, plugin_constants.SettingsType.COMMON)
+from edx_django_utils import plugins
+from openedx.core.djangoapps.plugins.constants import ProjectType, SettingsType
+INSTALLED_APPS.extend(plugins.get_apps(ProjectType.CMS))
+plugins.add_plugins(__name__, ProjectType.CMS, SettingsType.COMMON)
```

Comment:
Since we typically just import what we use, I would have expected:
```
from edx_django_utils.plugins import add_apps, add_plugins
```
Note: This also would have highlighted that `add_apps` should probably be `add_plugin_apps` to make it work better on its own.
Let's discuss to decide what way we should go with this.

Reply:
Yep, as you said, the names of some of the functions do not fully convey what is being done. Which is why I went with the idea of importing plugins themselves.

Reply:
Hmm, I've been hesitant to make changes to the plugins code itself, but I guess it just makes sense to just completely own that code and make the necessary improvements